pkg/operations/cordondrainvm.go

```diff
@@ -14,6 +15,10 @@ import (
 const (
 	interval            = time.Second * 1
 	mirrorPodAnnotation = "kubernetes.io/config.mirror"
+
+	// This is checked into K8s code but I was getting into vendoring issues so I copied it here instead
+	kubernetesOptimisticLockErrorMsg = "the object has been modified; please apply your changes to the latest version and try again"
```

Comment:
This seems like a brittle pattern match. (especially the `;`) Is just `"the object has been modified"` unique enough?

(Not to mention that this will need to be massaged over time as error messages change.)

Reply:
This is a constant in k8s code that we aren't currently vendoring when I did I ran into dependency conflicts. there wasn't a code I could of queued off of other wise I would of